### DISCUSSION TITLE
Bricked?

### DISCUSSION THREAD

**Discussion by K3vb3rt** [ludwig-v/wireless-carplay-dongle-reverse-engineering#42] [Q&A]
Hello guys,

just tried to upgrade my device via USB, but the stick won't be recognized. After that I updated via OTA but there I got the message "Update failed". Now the device is starting for a second and displays "Hardware Error: Wifi". 

What can I do? Is there any possibility to fix the issue via USB?

Thanks

**Comment by K3vb3rt** [comment]
Got it to start up with cable connected in Car. But when I plug off the cable there is the problem with WiFi screen. Seems to be WiFi-corrupted. 
i think would i be able to re-flash via USB it should be back. But the sticks are not accepted. Actually I’m on the screen for „Connection“ and connected via Bluetooth. The IP Adress is only available via Wifi 🤔

**Reply by MorgusVR** [reply]
Did you figure it out, mine just did this today.

**Reply by K3vb3rt** [reply]
Nope, no chance. Was still not able to reanimate the device. Even the seller is not accepting warranty or support.

**Comment by CiNcH83** [comment]
See [here](https://forums.macrumors.com/threads/carplay-adapter-to-convert-to-wireless-carplay.2193978/post-29736406). Some seem to have fixed the issue via rollback firmware. Guess the FW has to be flashed via USB thumb drive as WiFi isn't working!?
